src/presets/svelte/config-loader.ts

```diff
@@ -1,28 +1,28 @@
 // This file is a rewrite of `@sveltejs/vite-plugin-svelte` without the `Vite`
```

Comment:
I'd expect we can get rid of this file altogether now that we only support Vite, and can just get `loadSvelteConfig` from the Svelte Vite plugin, like we already do in the vite preset.